### PR TITLE
Rename Vantibyte video

### PR DESCRIPTION
**Pull request by Hiramiya**

### CODE REVIEW

File: _includes/footnote.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<span class="footnote"><div class="footnoteborder"></div>{{ include.text }}</span>
```
Comment (benolot) [inline _includes/footnote.html:1]:
```suggestion
<span class="footnote"><div class="footnoteborder"></div>{{ include.text }}</span>

```

File: _products/video.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Another option provided by our service is to have synchronised playback for all viewers, no matter where they are in the world. This allows all team members to be sure they're seeing the same content within a few milliseconds of eachother.
+
+## Example use case
+
+An esports production company has only one spectator slot available in their Rainbow Six Siege game. They need to be able to share the in game camera with their casters, desk analysts and of course, the audience all in real time.
+
+They get their in game observer to stream to {% include strongblue.html text="Vantibyte" %}'s Video solution. All other members on the team are able to watch it in real time, providing their commentary straight into the production desk in perfect sync with the game feed that Twitch.tv viewers see.
+
+{% include footnote.html text="The Video service by it's very nature uses a significant amount of bandwidth. Viewers with poor connections, or watching a large number of feeds, may experience difficulties." %}
```
Comment (benolot) [inline _products/video.md]:
```suggestion
{% include footnote.html text="The Video service by it's very nature uses a significant amount of bandwidth. Viewers with poor connections, or watching a large number of feeds, may experience difficulties. This is similar behaviour to other providers of live-video over IP such as TwitchTV and YouTube." %}
```

**Review by benolot — APPROVED** [review]
Left some optional feedback but LGTM

File: _includes/footnote.html (hunk 1 above)
Comment (Hiramiya) [reply _includes/footnote.html:1]: Nope - if we do this, then after every one of these it will render a newline into the completed HTML

**Pull request merged**